clio-client/src/main/scala/org/broadinstitute/clio/client/metadata/CramMover.scala

```diff
@@ -0,0 +1,288 @@
+package org.broadinstitute.clio.client.metadata
+import java.net.URI
+
+import org.broadinstitute.clio.client.dispatch.MoveExecutor.MoveOp
+import org.broadinstitute.clio.transfer.model.wgscram.{CramExtensions, CramMetadata}
+
+class CramMover extends MetadataMover[CramMetadata] {
+  override protected def moveMetadata(
+    src: CramMetadata,
+    destination: URI,
+    newBasename: Option[String]
+  ): (CramMetadata, Iterable[MoveOp]) = {
+    import MetadataMover.buildFilePath
+
+    val movedCram = src.cramPath.map(
+      buildFilePath(_, destination, newBasename.map(_ + CramExtensions.CramExtension))
+    )
+
+    val dest = src.copy(
+      cramPath = movedCram,
+      craiPath = movedCram.map { cramUri =>
+        // DSDEGP-1715: We've settled on '.cram.crai' as the extension and
+        // want to fixup files with just '.crai' when possible.
+        URI.create(s"$cramUri${CramExtensions.CraiExtensionAddition}")
+      },
+      logPath = src.logPath.map(buildFilePath(_, destination)),
```

Comment:
only `logPath` doesn't require handling `newBasename`?

Reply:
`logPath` and `readgroupLevelMetrics`. But `logPath` doesn't actually work as-is, so this should get removed for now.